lib/cib/cib_utils.c

```diff
@@ -229,7 +229,7 @@ cib__element_in_patchset(const xmlNode *patchset, const char *element)
 /*!
  * \brief Create XML for a new (empty) CIB
  *
- * \param[in] cib_epoch   What to use as "epoch" CIB property
+ * \param[in] cib_epoch  What to use as \c PCMK_XA_HAVE_QUORUM CIB attribute
```

Comment:
EPOCH

Reply:
How the heck...

Good catch among all the robotic review